Tests/PSModuleGallery.Type.Tests.ps1

```diff
@@ -886,4 +888,85 @@ InModuleScope 'PSDepend' {
+                $Dependency.Target | Should -Be $SavePath
+            }
+
+            It 'Installs the .NET Core SDK to the specified directory' {
+                Mock Test-Dotnet { return $false }
+
+                Invoke-PSDepend @Verbose -Path "$TestDepends\dotnetsdk.complex.depend.psd1" -Force
+                Test-Path $SavePath | Should -BeTrue
+            }
+
+            It 'Does nothing if the .NET Core SDK is found' {
+                Mock Test-Dotnet { return $true }
+                Mock Install-Dotnet 
+
+                Invoke-PSDepend @Verbose -Path "$TestDepends\dotnetsdk.complex.depend.psd1" -Force
+                Assert-MockCalled -CommandName Install-Dotnet -Times 0 -Exactly
+            }
+
+            AfterAll {
+                Remove-Item -Force -Recurse $SavePath -ErrorAction SilentlyContinue
+            }
+        }
+
+        Context 'Tests Dependency' {
+            # used to see if 'dotnet' is already on the PATH - we need this to return false
+            Mock Get-Command { return $false } -ParameterFilter { $Name -eq 'dotnet' }
+
+            It 'Can propertly compare semantic versions' {
+                Mock Test-Path { return $true } -ParameterFilter  { $Path -eq $GlobalDotnetSdkLocation }
+                Mock Get-DotnetVersion { return '2.1.330-rc1' }
+
+                # '2.1.330-rc1' >= '2.1.330-preview1'
+                # '2.1.330-rc1' >= '2.1.330-rc1'
+                # '2.1.330-rc1' >= '1.0'
+                Invoke-PSDepend @Verbose -Path "$TestDepends\dotnetsdk.semanticversion.depend.psd1" -Test -Quiet | Should -BeTrue
+            }
+        }
+
+        Context 'Imports Dependency' {
+            # used to see if 'dotnet' is already on the PATH - we need this to return false
+            Mock Get-Command { return $false } -ParameterFilter { $Name -eq 'dotnet' }
+
+            BeforeAll {
+                $originalPath = "$env:PATH"
+            }
+
+            It 'Can add the Target of the .NET Core SDK to the PATH' {
+                Mock Test-Dotnet { return $true }
+                Invoke-PSDepend @Verbose -Path "$TestDepends\dotnetsdk.complex.depend.psd1" -Force -Import -ErrorAction Stop
+                
+                $env:PATH | Should -Match "$SavePath*"
+            }
+            It 'Can add the global path of the .NET Core SDK to the PATH' {
+                Mock Test-Dotnet { return $true }
+                Invoke-PSDepend @Verbose -Path "$TestDepends\dotnetsdk.simple.depend.psd1" -Force -Import -ErrorAction Stop
+
+                $env:PATH | Should -Match "$globalDotnetSdkLocation*"
+            }
+            It 'Throws if the path cannot be found' {
+                Mock Test-Dotnet { return $false }
+                { Invoke-PSDepend @Verbose -Path "$TestDepends\dotnetsdk.simple.depend.psd1" -Force -Import -ErrorAction Stop } | Should -Throw `
+                    -ExpectedMessage ".NET SDK cannot be located. Try installing using PSDepend."
```

Comment:
nit: Could also be done as 

```powershell
{ Invoke-PSDepend @Verbose -Path "$TestDepends\dotnetsdk.simple.depend.psd1" -Force -Import -ErrorAction Stop } |
    Should -Throw -ExpectedMessage ".NET SDK cannot be located. Try installing using PSDepend."
```

I don't have anything against the back tick personally, but there's very few things that make the pitchforks come out quicker than back ticks 😉